src/dvclive/dvc.py

```diff
@@ -36,7 +36,10 @@ def make_checkpoint():
     if not root_dir:
         return
 
-    signal_file = os.path.join(root_dir, ".dvc", "tmp", env.DVC_CHECKPOINT)
+    tmp_dir = os.path.join(root_dir, ".dvc", "tmp")
+    os.makedirs(tmp_dir, exist_ok=True)
```

Comment:
Do we need this? `dvc.api.make_checkpoint` doesn't do it

Reply:
I think so - on dvc side we can be pretty sure that tmp is in place. I think that in `dvclive` we should not rely on it.
For example:
1. someone clones the repo - there is no `tmp` before first dvc command
2. someone runs training script without `dvc repro` - as cmd defined in dvc.yaml
3. script fails because `.dvc` exists and `.dvc/tmp` does not

Actually that might mean that we need to fix `dvc.api.make_checkpoint` too - seems like this use case is valid there too.

Reply:
> script fails because .dvc exists and .dvc/tmp does not

I think this would be a very edge case, it would require the user to manually set up the `env.DVC_CHECKPOINT`

Reply:
> I think this would be a very edge case, it requires the user to manually set up the env.DVC_CHECKPOINT

Good point, in case of exp run we will get the tmp.